### ISSUE TITLE
Optimizer loss function without desired paramater

### ISSUE THREAD

**Issue by nam15122013** [tensorflow/tensorflow#13173]
Hi all,
For Tensorflow, Is it possible to train  a neural network to optimize a loss function, e.g, minimize cost function, without a desired parameter? And what is optimizer in Tensorflow for the objective?
thank you

**Comment by shivaniag** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
